util/settings/settings.go

```diff
@@ -759,13 +759,13 @@ func (mgr *SettingsManager) GetServerRBACLogEnforceEnable() (bool, error) {
 func (mgr *SettingsManager) GetDeepLinks(deeplinkType string) ([]DeepLink, error) {
 	argoCDCM, err := mgr.getConfigMap()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error retrieving deep links %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error retrieving argocd-cm: %w", err)
```

The outer wrap will include the information that this is related to deep links.